components/fab/Fab.vue

```diff
@@ -6,7 +6,14 @@
     v-bind="$attrs"
     v-on="$listeners"
   >
-    <slot />
+    <slot name="icon" />
+    <div
+      v-if="extended"
```

Comment:
This seems to have introduced a bug which makes it impossible to make an extended FAB. The default slot is only shown if extended is true, but extended is set to false unless there is text in the default slot.

Reply:
Ahhh, I forgot to update readme. I'll open a new PR to fix it.